include/pclomp/ndt_omp.h

```diff
@@ -288,6 +298,21 @@ namespace pclomp
 			regularization_pose_ = boost::none;
 		}
 
+		NdtResult executeScanMatching(Eigen::Matrix4f initial_pose_matrix)
```

Comment:
When you pass an Eigen matrix to a function, you have to pass it by reference like `executeScanMatching(const Eigen::Matrix4f& initial_pose_matrix)`
Please check [this document](http://eigen.tuxfamily.org/dox/group__TopicPassingByValue.html)

Reply:
Resolved: https://github.com/tier4/ndt_omp/pull/16/commits/763211bddcfb6e328c81d517a104826a2fb3fc91 (no longer using that argment)